src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
 import {Dimensions, Pressable, ScrollView, StyleSheet, Text, View} from 'react-native';
-
 import {useRoute} from '@react-navigation/core';
-import PropTypes from 'prop-types';
+
+import {Context} from '../../../context';
 import {colors} from '../../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the order of these two imports has been swapped. Again, while this doesn't affect the functionality, maintaining a consistent order can improve readability.
```diff
- import {Context} from '../../../context';
- import {colors} from '../../../utils/colors';
+ import {colors} from '../../../utils/colors';
+ import {Context} from '../../../context';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->